src/main/scala/io/iohk/ethereum/network/p2p/messages/WireProtocol.scala

```diff
@@ -77,6 +78,21 @@ object WireProtocol {
       }
     }
 
+    object Reasons {
+      val DisconnectRequested = 0x00
```

Comment:
Minor detail, if we add variables for each of the codes for the reasons we should also remove them from the `MagicNumberChecker` ignore list from the scalastyle config file to keep it as small as possible.